R/library.R

```diff
@@ -82,10 +82,17 @@ install_package.local <- function(locked_package, libPath, quiet) {
 }
 
 install_package.tarball <- function(locked_package, libPath, quiet) {
-  stopifnot(is.element("url", names(locked_package)))
+  ## First, attempt to install the already downloaded tarball
+  try(utils::install.packages(locked_package$download_path, lib = libPath
+    , repos = NULL, type = "source", quiet = quiet))
```

Comment:
`silent=T`

Reply:
I do not see `silent` as one of the arguments to `utils::install.packages`. You mean somewhere else?